### PR TITLE
Bundle legacy stuff properly

### PR DESCRIPTION
**Pull request by Rich-Harris**
Alternative (and much less hacky!) fix for #9489 than https://github.com/mrdoob/three.js/pull/9518

### CODE REVIEW

File: src/Three.Legacy.js [hunk 1]
```diff
@@ -2,45 +2,86 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
-Object.assign( THREE, {
-	CubeGeometry: THREE.BoxGeometry,
```
Comment (mrdoob) [inline src/Three.Legacy.js]: I think you forgot `CubeGeometry`.

**Comment by mrdoob** [comment]
Much nicer indeed!

File: src/Three.Legacy.js (hunk 1 above)
Comment (Rich-Harris) [inline src/Three.Legacy.js]: whoops! I could have sworn... anyway, it's back in there now

**Comment by mrdoob** [comment]
Looks good now.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**